docs/installation/docker.md

```diff
@@ -6,20 +6,43 @@ title: Docker
 for every operating system on the [Docker website](https://docs.docker.com/engine/install/).**
 
 ## Quick Start
+prepare a folder where you are going to save your bbs files.
+- Generate some config for your BBS: \
+you can perform this step from anywhere - but make sure to consistently run it from the same place to retain your config inside the docker guest
+```
+docker run -it -p 8888:8888 \
+--name "EnigmaBBS" \
```

Comment:
Worth using the correct casing for `ENIGMA` for the container name?

Reply:
yes!